addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -671,6 +684,99 @@ var ImageWidget = SearchWidget.extend({
     },
 });
 
+/**
+ * Let users choose an image, including uploading a new image in odoo.
+ */
+var ImageWidget = FileWidget.extend({
+    template: 'wysiwyg.widgets.image',
+
+    /**
+     * @constructor
+     */
+    init: function (parent, media, options) {
+        options = options || {};
+        this._super.apply(this, [parent, media, _.extend({}, options, {
```

Comment:
Well it's not wrong... But I'll change of course, this was an adaptation from `this._super.apply(this, arguments);` where `arguments` was changed, too focused on adapting it to notice it could be simpler.